src/Request.php

```diff
@@ -24,7 +24,7 @@ class Request implements RequestInterface
     use MessageTrait, RequestTrait;
 
     /**
-     * @param null|string $uri URI for the request, if any.
+     * @param null|string|UriInterface $uri URI for the request, if any.
```

Comment:
Has been that way since [1.0.0](https://github.com/zendframework/zend-diactoros/blob/1.0.0/src/RequestTrait.php#L68-L80).